keyboards/enviousdesign/60f/config.h

```diff
@@ -0,0 +1,154 @@
+/* Copyright 2022 Adam K (@Envious-Data)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xE739
+#define PRODUCT_ID 0xA400
+#define DEVICE_VER 0x0001
+#define MANUFACTURER EnviousDesign
+#define PRODUCT 60F 1.0
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS \
+    { 11, 12, 13, 14, 15 }
+#define MATRIX_COL_PINS \
+    { 10, 9, 8, 7, 6, 5, 16, 17, 18, 19, 20, 21, 22, 26, 27 }
```

Comment:
These all should probably have the `GP` prefix to match the actual pin abstractions.